### ISSUE TITLE
Failed to render chord in Jupyter notebook

### ISSUE THREAD

**Issue by gramster** [Abjad/abjad#1132]
Because of #1128 I installed abjad from the repo head:

```
pip install ".[ipython,development]"
```

I'm in a Python 3.7 virtual environment on MacOS Mojave and have lilypond available:

```
(py37) ➜  abjad git:(master) lilypond
GNU LilyPond 2.18.2
Usage: lilypond [OPTION]... FILE...
```

I create a notebook, and then try:

```
%load_ext abjadext.ipython
import abjad
```

and:

```
chord = abjad.Chord("<ef' f' cs''>4")
abjad.show(chord)
```

But that results in:

```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-3-66358083902d> in <module>
      1 chord = abjad.Chord("<ef' f' cs''>4")
----> 2 abjad.show(chord)

~/anaconda/envs/py37/lib/python3.7/site-packages/abjadext/ipython/Show.py in __call__(self, argument)
     27             pngs = []
     28             if not success:
---> 29                 raise RuntimeError('Rendering failed')
     30             if not png_paths:
     31                 raise FileNotFoundError('LilyPond PNG output not found.')

RuntimeError: Rendering failed
```

**Comment by zepadovani** [comment]
Similar problem here with Abjad 3.1, LilyPond 2.19.83 and ImageMagick 6.9.9-40

`%load_ext abjadext.ipython
import abjad`

`note = abjad.Note(0,1/4)
abjad.show(note)`

gives me:
`---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-10-d04a6689cdc6> in <module>
      1 note = abjad.Note(0,1/4)
----> 2 abjad.show(note)

/usr/local/lib/python3.7/site-packages/abjadext/ipython/Show.py in __call__(self, argument)
     27             pngs = []
     28             if not success:
---> 29                 raise RuntimeError('Rendering failed')
     30             if not png_paths:
     31                 raise FileNotFoundError('LilyPond PNG output not found.')

RuntimeError: Rendering failed`

I have noticed that a pdf of the score is being created inside the working directory, and I suppose that the "show" method is not calling the right backend when executing LilyPond.

**Comment by bitzl** [comment]
Any news on this? Or any way I could help? Using Abjad in a Jupyter notebook would be my main usecase.

**Comment by trevorbaca** [comment]
Fixed in Abjad 3.2 (released 2021-01-19).
